### PR TITLE
location code change

### PR DESCRIPTION
**Pull request by elidana**
Hi,

this is a first draft of the communication around Strong Motion Location code change. 

The update is:
- update readme file for the geonet/help repository
- add a "folder" that can be used to list significant changes in data and metadata
- add a markdown file for each of those changes

This pull request also contains a placeholder (`component-name.md`) for other changes in the seismic data. This still need to be worked on.

Happy to receive any feedback!

Thanks

### CODE REVIEW

**Comment by salichon** [comment]
Hello @elidana  this draft is looking good for a start  thanks a lot.
As much as possible we should add a time tag for each change listing with the date it was implemented in the archive. 
cheers!
jerome

File: data-changes/location-codes.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+For those SM stations, location codes that were previously recorded as `20` are now being changed to `21`.
+
+Due to the complexity of this work, there have been for a period of time a mismatch between the metadata (that were correctly reporting 21) and the data in the archive. This has now been fixed.
+
+### Affected data
+
+Data and metadata that are affected by this change are:
+- strong motion metadata (distributed via delta, FDSN)
+- strong motion miniseed data (distributed via FDSN)
+- _###TODO: list data products?_
```
Comment (mnaguit) [inline data-changes/location-codes.md:37]: add: strong motion processed data

**Comment by elidana** [comment]
Hi @mnaguit and  @salichon , thanks for your review!

@salichon will try my best to add the time when change was applied, though if we are applying all changes within 2-3 months, can we just say that?

File: data-changes/location-codes.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+Due to the complexity of this work, there have been for a period of time a mismatch between the metadata (that were correctly reporting 21) and the data in the archive. This has now been fixed.
+
+### Affected data
+
+Data and metadata that are affected by this change are:
+- strong motion metadata (distributed via delta, FDSN)
+- strong motion miniseed data (distributed via FDSN)
+- strong motion processed data (distributed via the Strong Motion Tool and archive)
+- seismic data picks ID and magnitude ID (distributed via FDSN) might have metadata not correctly associated
```
Comment (mnaguit) [inline data-changes/location-codes.md:37]: what are seismic data picks ID and magnitude ID?

Comment (elidana) [reply data-changes/location-codes.md:37]: that was a comment from @salichon , I understand those are metadata available in the quakeml ? if there is a better description of that I'd really appreciate suggestions!

Comment (mnaguit) [reply data-changes/location-codes.md:37]: No idea at all. Just curious to know what those IDs are. The only ID I'm aware of is the event public ID.

**Comment by elidana** [comment]
Hi @mnaguit @salichon @rumachan , are we happy to merge this one (possibly without the "component change one, that I can strip out and add in a second pull request) or do you think we need to apply other changes?
Thanks!!

**Comment by rumachan** [comment]
Yep, lets make progress and add other stuff later if needed.

**Comment by elidana** [comment]
Thanks all, I have tidy up this pr, and added all your comments (hopefully).
Can I please have one thumbs up, so this one can be merged? Thanks a lot!

**Review by salichon — CHANGES_REQUESTED** [review]
- in `readme`: link "Bug writing guidlines" shoudl be  "Bug writing guidelines"

- In `data changes readme`  i would say add or mention  "archived data" changes 
as opposed to real time data that are rather "corrected" on the fly in general quickly (and then avoiding us to correct the archive ...) 
Also we re (in this procedure) ionly modifying the internal Miniseed file headers of the archived data and file name to fit our standards of orienation and instrument location. 
**and not the Data/measures per se**  [bad data is bad data good remains good] 
This may imply a Metadata repository correction.  
The data in the archive are corrected (now) only if an issue was not identified before about one week.
Hence  we may  have a significant backlog with the past recordings of the last 20+ years.    
 
 - in `location-codes.md`  _naming requirements for Seismohraph Stations_  should be _naming requirements for Seismograph Stations_
- I think adding the type of instrument affected by the data change will be useful specially for multi instrumented stations

**Comment by salichon** [comment]
@elidana as Steve  about merging "_Yep, lets make progress and add other stuff later if needed_."

so that we can finalize the recent archive update

**Review by salichon — APPROVED** [review]

**Pull request merged**